engine/src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -7,16 +7,19 @@ import cromwell.engine.ExecutionStatus
 import cromwell.engine.ExecutionStatus.ExecutionStatus
 import cromwell.engine.db.ExecutionDatabaseKey
 import cromwell.engine.workflow.BackendCallKey
+import slick.dbio.Effect.Read
+import slick.jdbc.GetResult
 import slick.profile.RelationalProfile.ColumnOption.Default
 import wdl4s.{NamespaceWithWorkflow, Scatter}
 
 import scala.language.postfixOps
-import scala.util.{Success, Failure, Try}
+import scala.util.{Failure, Success, Try}
 
 case class CallCacheHit(workflowId: String, callName: String)
 case class ExecutionWithCacheData(execution: Execution, cacheHit: Option[CallCacheHit])
 
-case class Execution(workflowExecutionId: Int,
+case class Execution(executionId: Option[Int] = None,
```

Comment:
I'll allow it I as this class is already borked, but in general I'm not a fan of default arguments before non-default. I'm not quite sure how and when this class ended up with this hybrid approach. Having defaults before non-defaults forces the caller to use named arguments _only_, i.e. no short form `Execution("fqn", 1, "Starting")`